bash.md

```diff
@@ -481,6 +481,7 @@ History
 
 ### Operations
 
+| `sudo !!` | Execute last command as root |         
```

Comment:
I don't think we should have any `sudo` commands in here, so I'd like to remove this one. The other changes are good!